manifests/s/Sky/SkyGo/21.8.2.0/Sky.SkyGo.installer.yaml

```
@@ -0,0 +1,18 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Sky.SkyGo
+PackageVersion: 21.8.2.0
+Installers:
+- MinimumOSVersion: 10.0.0
+  Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://desktopclient.ott.sky.com/skygodesktop/GB/SkyGoInstaller.exe
+  InstallerSha256: C8FC300C3FECEB48D16868005DD779E8C4273A19B941AE98B91FA14D62E94E8F

```

```suggestion
  InstallerSha256: C8FC300C3FECEB48D16868005DD779E8C4273A19B941AE98B91FA14D62E94E8F
  ProductCode: "com.bskyb.skygoplayer_is1"
```

To improve matching.